<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoLayoutOptionPage</class>
 <widget class="QWidget" name="AutoLayoutOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>270</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Layout Generator</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="m_autoDotPath">
        <property name="toolTip">
         <string>&lt;p&gt;The auto layout feature depends on layout generators provided by the GraphViz package, which is normally installed aside Umbrello by a package manager.&lt;/p&gt;
&lt;p&gt;Umbrello has built in support for detecting the install location of  these layout generators. For cases where this dependency is not available or do not fit, a different installation path could be provided.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Auto detect layout generator</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Generator executable path:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="KUrlRequester" name="m_dotPath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="mode">
         <set>KFile::Directory|KFile::ExistingOnly</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Other">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="m_showExportLayout">
        <property name="toolTip">
         <string>&lt;p&gt;Dot file export is performed by using the export layout.&lt;/p&gt;&lt;p&gt;With this option checked, the export layout is added to the list of available diagram layouts and enables a quick dot export preview.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Show export layout in diagram context menu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_autoDotPath</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_dotPath</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
